core/core-configuration-layer.el

```diff
@@ -2694,6 +2693,13 @@ MSG is an additional message append to the generic error."
            configuration-layer-stable-elpa-name
            reason)))
 
+;; autoload functions from epg package
+(autoload 'epg-context-home-directory "epg")
+(autoload 'epg-import-keys-from-file "epg")
+(autoload 'epg-verify-file "epg")
+(autoload 'epg-context-result-for "epg")
+(autoload 'epg-signature-status "epg")
+
 (defun configuration-layer//stable-elpa-verify-archive ()
   "Verify the downloaded stable ELPA repository archive.
 
```

Comment:
Can we just do `(require 'epg)` in the first line of `configuration-layer//stable-elpa-verify-archive` instead?  It would let us avoid listing a whole bunch of functions that may go out of sync easily.

Reply:
@bcc32  No, require 'epg in the function does not work for this case, it caused by the `epg-verify-file` is an eieio created function for accessing the epg structure member, which is not defined during `defun`,
Testing commands:
```
emacs -batch -l ert -l ~/.emacs -l tests/core/core-configuration-layer-ftest.el -f ert-run-tests-batch-and-exit
```
> Test test-stable-elpa-verify-archive--verification-failed backtrace:
    (void-function \(setf\ epg-context-home-directory\))